source/module_base/module_mixing/plain_mixing.h

```diff
@@ -108,12 +109,36 @@ class Plain_Mixing : public Mixing
 
         // container::Tensor data = data_in + mixing_beta * F;
         std::vector<FPTYPE> data(length);
+        if (GlobalV::NSPIN == 1 || GlobalV::NSPIN == 4)
+        {
+            // rho_tot
 #ifdef _OPENMP
 #pragma omp parallel for schedule(static, 4096 / sizeof(FPTYPE))
 #endif
-        for (int i = 0; i < length; ++i)
+            for (int i = 0; i < length; ++i)
+            {
+                data[i] = data_in[i] + this->mixing_beta * F_tmp[i];
+            }
+        }
+        else if (GlobalV::NSPIN == 2)
```

Comment:
Mix_matrix also uses mixing class. In that case, maybe it is wrong.

Reply:
How about adding two input array, one contains `alpha`, another one contains corresponding `index_start`?

Reply:
You can use a different global varible temporarily. Maybe TWO_BETA ...
In the future, I will refactor push_data function to make it more general.

Reply:
Done.